### PR TITLE
PLT-3840 Fix Typos in cluster localization strings

### PR DESCRIPTION
**Pull request by enahum**
#### Summary
Fixes a typo en cluster localization

#### Ticket Link
https://mattermost.atlassian.net/browse/PLT-3840

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] Added or updated unit tests (required for all new features)
- [ ] Added API documentation (required for all new APIs)
- [ ] All new/modified APIs include changes to the drivers
- [x] Has enterprise changes (https://github.com/mattermost/enterprise/pull/50)
- [ ] Has UI changes
- [x] Includes text changes and localization files updated
- [ ] Touches critical sections of the codebase (auth, upgrade, etc.)

### CODE REVIEW

File: i18n/en.json [hunk 1]
```diff
@@ -4708,12 +4708,12 @@
     "translation": "Cluster ping successful with hostname=%v on=%v with id=%v self=%v"
   },
   {
-    "id": "ent.cluster.incompatibile.warn",
```
Comment (jasonblais) [inline i18n/en.json]: I'm assuming changing the "id" won't have an effect elsewhere in the code?

**Comment by jasonblais** [comment]
+1 other than note above

**Comment by enahum** [comment]
actually it does, thanks I'll fix that

2016-08-08 13:53 GMT-05:00 Jason Blais <notifications@github.com>:

**Comment by crspeller** [comment]
+1 Assuming the ID was the typo :)

**Comment by crspeller** [comment]
+1 Other than the one fix.

**Comment by jwilander** [comment]
+1 other than one fix

**Pull request merged**